src/cuda/image_function_cuda.cu

```diff
@@ -43,6 +44,16 @@ namespace
     const FunctionRegistrator functionRegistrator;
 
     // The list of CUDA device functions on device side
+    __global__ void normalizeCuda( const uint8_t * image, uint32_t rowSize, bool horizontal, uint32_t width, uint32_t height, uint32_t * projection )
```

Comment:
Please put this function at the place to have alphabetic order among all functions

Reply:
Why is it called `normalize` function?

Reply:
changing the function name.

Reply:
It should be called something like `projectionProfileCuda`.